api/api/experiments.py

```diff
@@ -149,9 +150,11 @@ class Experiment(sqlm.SQLModel, table=True):  # type: ignore[call-arg]
     runner: runners.Runner = sqlm.Relationship(back_populates="experiments")
     sink_id: int = sqlm.Field(foreign_key="sink.id")
     sink: sinks.Sink = sqlm.Relationship(back_populates="experiments")
+    task_ids: str | None = sqlm.Field(default=None)
```

Comment:
I think a cleaner way than using `str` is to store the IDs in a list; see [here](https://github.com/tiangolo/sqlmodel/issues/178)

Reply:
thanks, i was looking for that. 
I tried `task_ids: List[str]` and variations of that but was not working.